src/main/resources/l10n/JabRef_en.properties

```diff
@@ -1,297 +1,159 @@
 Unable\ to\ monitor\ file\ changes.\ Please\ close\ files\ and\ processes\ and\ restart.\ You\ may\ encounter\ errors\ if\ you\ continue\ with\ this\ session.=Unable to monitor file changes. Please close files and processes and restart. You may encounter errors if you continue with this session.
 %0\ contains\ the\ regular\ expression\ <b>%1</b>=%0 contains the regular expression <b>%1</b>
-
```

Comment:
Please revert the deletion of the blank lines, just add the new keys at the bottom